Website/DesktopModules/Hotcakes/MiniCart/MiniCartView.ascx

```diff
@@ -1 +1,5 @@
 ﻿<%@ Control Language="C#" AutoEventWireup="true" CodeBehind="MiniCartView.ascx.cs" Inherits="Hotcakes.Modules.MiniCart.MiniCartView" %>
+<%@ Register TagPrefix="dnn" TagName="MiniCartSpa" Src="../MiniCartSkinObject/MiniCartView.ascx" %>
+
+
+<dnn:MiniCartSpa runat="server" ViewName="<% ViewName %>" id="MiniCartSpa" />
```

Comment:
@ismaelTartola This appears to be the same control that's being used for the module.  Is that correct?  Can both the module and the skin object be on the same page at the same time?  Does the module still work independently, the same as it did before?

Reply:
@WillStrohl It's not. What you are noticing here is just a pure naming. I named the main control inside of the skinOject folder just as the main control of the Minicart Module (Of course, they are placed in different folders). I addition to that I like to point that, now the Minicart module itself it's using the skin object, so if we merge this changes every time the Minicart module is used the skin object is used (HCC -> Minicart Module -> Minicart Skin object -> Minicart SPA view). To the question of if the module and the skin object can be used at the same time in a page the answer is YES, no problems there, they will work individually.

Reply:
@ismaelTartola Okay... Just so I'm sure I understand, both the SPA skin object and the module are using the same underlying code, but the UI is initiated from different UI/UX locations.   Is that what you're saying?